misc/docs/source/tutorial/code_snippets/waveform_plotting_tutorial_6.py

```diff
@@ -32,6 +34,19 @@
     tr.stats.network = 'TOK'
 
 st.filter('bandpass', freqmin=0.1, freqmax=10)
-# Plot
+
+# Do the section plot..
+# If no customization is done after the section plot command, figure
+# initialization can be left out and also option ".., show=False, fig=fig)" can
+# be omitted, and figure is shown automatically
+fig = plt.figure()
 st.plot(type='section', plot_dx=20e3, recordlength=100,
-        time_down=True, linewidth=.25, grid_linewidth=.25)
+        time_down=True, linewidth=.25, grid_linewidth=.25, show=False, fig=fig)
```

Comment:
Instead of `show=False, fig=fig`, you could do `fig = st.plot(..., handle=True)` since you don't really do anything with the figure before hand.